erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -45,6 +46,12 @@ def validate_delivery_date(self):
 			for location in self.locations if location.get("sales_order_item")]
 		self.delivery_date = min(order_delivery_dates)
 
+	def validate_stock_qty(self):
+		for row in self.locations:
+			order_qty = frappe.db.get_value("Sales Order Item", row.get("sales_order_item"), "qty")
```

Comment:
Check if `sales_order_item` exists before doing this request. And is there a different doctype that can be connected to Pick List, and should we have this check for those ones as well?